aten/src/THC/THCTensorMathReduce.cuh

```diff
@@ -420,6 +421,7 @@ THC_transformReduceOuterDimIndex(THCState *state,
   THCudaCheck(cudaGetLastError());
 }
 
+
 /* Reduce the innermost dimension of a tensor (on thrust::pair functors which are (value, index))
  *
  * For an n-d tensor (n <= 4) where the reduction is along the innermost dimension:
```

Comment:
From @wesolwsk: can we assume anything about the size of the innermost dimension compared to the other dimensions? Optimal implementations will be different depending on which is larger.